### PR TITLE
feat: add monsieurbiz_sylius_sales_reports_plugin as recipe

### PR DESCRIPTION
**Pull request by jovuk**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: monsieurbiz/sylius-sales-reports-plugin/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "MonsieurBiz\\SyliusSalesReportsPlugin\\MonsieurBizSyliusSalesReportsPlugin": [
```
Comment (toofff) [inline monsieurbiz/sylius-sales-reports-plugin/1.0/manifest.json:3]: `"MonsieurBiz\\SyliusSalesReportsPlugin\\MonsieurBizSyliusSalesReportsPlugin": ['all' => true]`

**Review by toofff — APPROVED** [review]
After small fix

File: monsieurbiz/sylius-sales-reports-plugin/1.0/manifest.json (hunk 1 above)
Comment (jacquesbh) [reply monsieurbiz/sylius-sales-reports-plugin/1.0/manifest.json:3]: Actually the recipe is correct. `=> true` is for the `bundles.php` in a symfony application, not in the manifest.

**Review by dunglas — APPROVED** [review]
The Travis failure doesn't look related to the recipe.

**Pull request merged**